deploy/helm/sumologic/conf/cleanup/cleanup.sh

```diff
@@ -10,7 +10,7 @@ export NO_PROXY=${NO_PROXY:=""}
 cp /etc/terraform/*.tf /terraform/
 cd /terraform || exit 1
 
-terraform init
+terraform init -input=false -get=false || terraform init -input=false -upgrade
```

Comment:
```suggestion
# Fall back to init -upgrade to prevent:
# Error: Inconsistent dependency lock file
terraform init -input=false -get=false || terraform init -input=false -upgrade
```